domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -17,6 +18,7 @@
 
 // Create these fields in JSON when serialize Java object, ignore them when deserialize.
 @JsonIgnoreProperties(value = {"totalAmountTillToday", "totalAmountTillDateOfIssue"}, allowGetters = true)
+@Builder
```

Comment:
not sure how I feel about lombok being added here =/

Reply:
@rlewan @geek-soft

Reply:
I would prefer to not add Lombok if it's not really needed. `Claim` constructor is quite big but we probably need to invoke it in only one place in the code.

Reply:
In my opinion, the question is not `lombok`, but do we need a `Builder` for this class? It shouldn't matter whether we choose over the shelf or write our own `Builder`. 
Whether it is invoked from single place or multiple, it leaves smell in the code.I would prefer not to call a constructor of class where I have to pass null for more than half of the parameters. Please advise if I am missing anything.

Reply:
Good point @sabahirfan. In that case I would say I'm ok with having the builder, my worry was that adding Lombok would incentivise people to make `Claim` a mutable object (I hope it doesn't generate any setters in this setup).

Reply:
No it's not generating setters just the Builder.